www/index.html

```diff
@@ -18,17 +19,17 @@
             <img class="spinner" src="img/binary-symbol-logo.svg" />
         </div>
     </div>
-
-<script src="boot.js"></script>
-<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=fetch,Promise,Array.from,Array.prototype.find,Array.prototype.includes,String.prototype.startsWith,String.prototype.includes,Math.sign,Intl.~locale.en,Intl.~locale.id,Intl.~locale.ru,Intl.~locale.fr&unknown=polyfill"></script>
-<script async src="app.js"></script>
-
+<script type="text/javascript" src="cordova.js"></script>
```

Comment:
@borisyankov we can separate the two then . I added a home.html for the cordova .